### PR TITLE
Updated sources of documentation

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Updated documentation skeletons
- Naming of files were applied as generated.
- Sphinx v6 is used (probably, 7 will be different)

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] CI Checkers workflow

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+The directory is used for documentation of CredSweeper with using [sphinx](https://www.sphinx-doc.org/en/master/),
+
+## Workflow
+
+There is applied custom documentation, so auto-generation might fail. Please, use auto-generation as reference.
+With the command new sources might be updated (in /docs directory):
+
+```bash
+sphinx-apidoc --force --full --ext-autodoc ../credsweeper -o source/
+```
```
Comment (csh519) [inline docs/README.md:12]: What about add those command to github actions?
It can be a good test of document i think.

**Review by csh519 — APPROVED** [review]
Thank you for adding `README.md` of docs and updates.

LGTM :+1:

File: docs/README.md (hunk 1 above)
Comment (babenek) [reply docs/README.md:12]: The command does not generate correct .rst files for current config (custom main page and so on). 
Maybe in next PR...

**Review by Dmitriy-NK — APPROVED** [review]
Looks good. Thanks for updating the documentation

File: docs/README.md (hunk 1 above)
Comment (csh519) [reply docs/README.md:12]: Okay. Thank you!

**Pull request merged**